cmd/skopeo/inspect.go

```diff
@@ -187,10 +201,31 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 			logrus.Warnf("Registry disallows tag list retrieval; skipping")
 		}
 	}
-	out, err := json.MarshalIndent(outputData, "", "    ")
-	if err != nil {
+	if len(opts.format) == 0 {
+		out, err := json.MarshalIndent(outputData, "", "    ")
+		if err == nil {
+			fmt.Fprintf(stdout, "%s\n", string(out))
+		}
 		return err
 	}
-	fmt.Fprintf(stdout, "%s\n", string(out))
-	return nil
+	return outputFormat(outputData, opts.format)
+}
+
+func outputFormat(data interface{}, format string) error {
+	var out formats.Writer
+	if format == "json" || format == "" { // "" for backwards compat
```

Comment:
We don’t need the `""` case.

Reply:
Since we don't have backwards compatibility I am removing this check altogether.